it/controllers/IndexControllerISpec.scala

```diff
@@ -1234,9 +1232,7 @@ class IndexControllerISpec extends IntegrationSpecCommonBase {
         val request = controller.onPageLoad()(fakeAgentRequest)
         await(request).header.status shouldBe Status.OK
         val parsedBody = Jsoup.parse(contentAsString(request))
-        parsedBody.select("#late-submission-penalties p.govuk-body").get(0).text shouldBe "Your client has 1 penalty point. This is because:"
-        parsedBody.select("#late-submission-penalties ul li").get(0).text shouldBe "they have submitted 2 VAT Returns late"
-        parsedBody.select("#late-submission-penalties ul li").get(1).text shouldBe "we removed 1 point and sent them a letter explaining why"
+        parsedBody.select("#late-submission-penalties p.govuk-body").get(0).text shouldBe "No active late submission penalty points."
```

Comment:
I don't think this is right. If you revert it back to 1 active point and then do the following:

> Seems like this has caused some debate. In the meantime, could you set one of those points in details to be an active point (penaltyStatus = LSPPenaltyStatusEnum.Active, FAPIndicator = None)

Reply:
Sure, reverted back.